include/circt/Support/FVInt.h

```diff
@@ -543,6 +547,32 @@ class FVInt {
     return v;
   }
 
+  /// Compute an unsigned division.
```

Comment:
Do you have references for these semantics? Should div-by-zero return X value?

Reply:
I've added a reference to the SV spec. Good point about the div-by-zero producing X. I forgot about that. Added a test for it 👍

Reply:
Cool, thanks!